manim/mobject/opengl_mobject.py

```diff
@@ -614,7 +615,13 @@ def func(points):
     def rotate_about_origin(self, angle, axis=OUT):
         return self.rotate(angle, axis, about_point=ORIGIN)
 
-    def rotate(self, angle, axis=OUT, **kwargs):
+    def rotate(
+        self,
+        angle,
+        axis=OUT,
+        about_point: Union[np.ndarray, List, None] = None,
```

Comment:
I've now added this parameter to opengl as well, but I have  never worked on it and don't know how to test it /if there are any tests. @eulertour : can you confirm that this is correct?

Reply:
It won't break anything if that's your concern

Reply:
Okay, perfect, yes that was the concern.